pkg/tfgen/pluginHost.go

```diff
@@ -122,7 +138,7 @@ func (host *inmemoryProviderHost) Provider(pkg tokens.Package, version *semver.V
 func (host *inmemoryProviderHost) ResolvePlugin(kind apitype.PluginKind, name string,
 	version *semver.Version) (*workspace.PluginInfo, error) {
 	if name == host.provider.name.String() {
-		info, err := host.provider.GetPluginInfo()
+		info, err := host.provider.GetPluginInfo(context.TODO())
```

Comment:
`context.Background()` is also an option here, potentially -- thoughts? I'm not sure if it's possible to change the signature of this method/what the implications of that are, but that's also an option.